app/views/admin/users/new.html.erb

```diff
@@ -38,10 +38,6 @@
 
                   <%= simple_form_for @user, url: admin_users_path do |f| %>
                     <div class="form-inputs">
-                      <%= f.input :organization_id, label: "Organization", wrapper: :input_group, required: true, autofocus: true do %>
-                        <span class="input-group-text"><%= fa_icon "building" %></span>
-                        <%= f.input_field :organization_id, collection: @organizations, class: "form-control" %>
-                      <% end %>
```

Comment:
The ticket suggested not making changes to New, just Edit.

Reply:
This is true.  I'm trying to find the issue that past CL failed to link to in this issue.